### DISCUSSION TITLE
Can't load the 3D print file

### DISCUSSION THREAD

**Discussion by fpd62** [GyroGearl00se/ESPHome-Froeling-Lambdatronic_3200#21] [General]
Hallo,

the 3D printing file wemos_esp32_rs232.3mf doesn't load in my case. Would you please be so kind and check the file?
Can you also provide a stl file?

Thanks

**Comment by GyroGearl00se** [comment]
Hi @fpd62 ,

I just uploaded it as a STL file. Additionally I've uploaded the f3d file that you open it in Fusion 360 and edit it easier if needed.
I'm new with designing parts for 3D printing. It is probably not the best design for this case - But it'll sort of do the job :)
